script/helpers/common.ts

```diff
@@ -54,8 +55,9 @@ export const updateConnectorStatus = async (
         siblingConnectorAddresses[it];
       if (!itConnectorAddress) continue;
 
-      let currentConnectorStatus =
-        await bridgeContract.callStatic.validConnectors(itConnectorAddress);
+      let currentConnectorStatus = getDryRun()
+        ? false
```

Comment:
this is only static call right ? why need to check dryRun here?

Reply:
If not handled this way, the call will log and throw an exception:

<img width="1184" alt="Screenshot 2024-06-12 at 12 51 41" src="https://github.com/SocketDotTech/socket-plugs/assets/69167058/784b0658-a372-45df-9d4d-2321a416fa8d">